spec/services/idv/profile_maker_spec.rb

```diff
@@ -1,21 +1,30 @@
 require 'rails_helper'
 
-describe Idv::ProfileFromApplicant do
-  describe '#create' do
+describe Idv::ProfileMaker do
+  describe '#new' do
     it 'creates Profile with encrypted PII' do
       applicant = Proofer::Applicant.new first_name: 'Some', last_name: 'One'
       normalized_applicant = Proofer::Applicant.new first_name: 'Somebody', last_name: 'Oneatatime'
       user = create(:user, :signed_up)
       user.unlock_user_access_key(user.password)
-      profile = described_class.create(
+
+      profile_maker = described_class.new(
         applicant: applicant,
         user: user,
         normalized_applicant: normalized_applicant
       )
 
+      profile = profile_maker.profile
+      pii = profile_maker.pii_attributes
+
+      expect(profile).to be_a Profile
       expect(profile.id).to_not be_nil
       expect(profile.encrypted_pii).to_not be_nil
       expect(profile.encrypted_pii).to_not match 'Some'
+
+      expect(pii).to be_a Pii::Attributes
+      expect(pii.first_name.raw).to eq 'Some'
+      expect(pii.first_name.norm).to eq 'Somebody'
```

Comment:
Those methods are both just accessors. They don't *do* anything. It's really `new` that requires testing.